.rubocop_todo.yml

```diff
@@ -500,11 +464,6 @@ Style/RegexpLiteral:
 Style/RescueEnsureAlignment:
   Enabled: false
 
-# Offense count: 46
```

Comment:
Really? https://github.com/bbatsov/ruby-style-guide#no-rescue-modifiers 

This introduces more leeway for hiding exceptions under `rescues` that catch too much